eventarc/generic/main.go

```diff
@@ -62,11 +65,10 @@ func GenericHandler(w http.ResponseWriter, r *http.Request) {
 		Headers: headerMap,
 		Body:    body,
 	}
-	out, err := json.Marshal(res)
-	if err != nil {
-		panic(err)
+	if err := json.NewEncoder(w).Encode(res); err != nil {
```

Comment:
I'm a bit confused -- why write the body and heads above _and_ in JSON here? I might be missing something.

Reply:
Thanks for catching. You're right, we already respond here so no need to respond above.

Simplified the sample a bit.